src/main/java/no/unit/nva/cristin/projects/PresentationConverter.java

```diff
@@ -42,11 +50,23 @@ protected ProjectPresentation asProjectPresentation(Project project, String lang
             fundingSourcePresentation.fundingSourceCode = fundingSource.fundingSourceCode;
             fundingSourcePresentation.projectCode = fundingSource.projectCode;
             fundingSourcePresentation.names = fundingSource.fundingSourceName.entrySet().stream()
-                    .map(name -> new FundingSourceNamePresentation(name.getKey(), name.getValue()))
-                    .collect(Collectors.toList());
+                .map(name -> {
+                    FundingSourceNamePresentation fundingSourceNamePresentation =
+                        new FundingSourceNamePresentation();
+                    fundingSourceNamePresentation.language = name.getKey();
+                    fundingSourceNamePresentation.name = name.getValue();
+                    return fundingSourceNamePresentation;
+                })
+                .collect(Collectors.toList());
             projectPresentation.fundings.add(fundingSourcePresentation);
         });
 
+        //String input = (attempt(() -> (JsonUtils.objectMapper.writeValueAsString(project))).get());
+        //logger.info(input);
+
+        //String output = (attempt(() -> (JsonUtils.objectMapper.writeValueAsString(projectPresentation))).get());
+        //logger.info(output);
+
```

Comment:
I will remove these commented lines in this class as I don't need them for testing anymore